R/pkg/inst/tests/test_binaryFile.R

```diff
@@ -27,7 +27,7 @@ test_that("saveAsObjectFile()/objectFile() following textFile() works", {
   fileName2 <- tempfile(pattern="spark-test", fileext=".tmp")
   writeLines(mockFile, fileName1)
 
-  rdd <- textFile(sc, fileName1)
+  rdd <- textFile(sc, fileName1, 1)
```

Comment:
Not related to this PR but the `1` vs. `1L` thing (i.e. numeric vs. int) should also be standardized across all functions (they should accept numeric)